webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -30,25 +33,69 @@ export default class GithubRepoSelector extends PureComponent {
     }
 
     componentDidMount() {
-        this.props.actions.getRepos();
+        this.props.actions.getOrgs(true);
```

Comment:
Sorry I thought I commented this before but I guess not. Do we need this `true` passed in here? It's a bit cryptic, and I believe this is the only spot we call this function. Thoughts on getting rid of the parameter?

Reply:
Yes, this is only called here. The `true` parameter is for `includeLoggedInUser`. If we pass this flag as true, then only the server will send a `username` as part of the organization's list. I thought the `getOrgs` action would be reusable in the future, so it would be better to have this flag as a parameter. What are your thoughts on this?

Reply:
@thoratvinod Not sure about this. Possibly [yagni](https://martinfowler.com/bliki/Yagni.html), and the cryptic nature of it makes it a bit harder to read. I'm inclined to remove the flag. I'm not sure if we ever _don't_ want that when the org config is not set.

Reply:
Done.